datahub-web-react/src/conf/theme/theme_dark.config.json

```diff
@@ -27,6 +27,15 @@
         },
         "search": {
             "searchbarMessage": "Search Datasets, People, & more..."
+        },
+        "menu": {
+            "items": [
+                {
+                    "label": "Datahub Wiki",
```

Comment:
should probably change this label name... Datahub Wiki isn't accurate ... not making this a showstopper.